src/components/OrderSidebar.js

```diff
@@ -23,6 +23,11 @@ const OrderSidebar = () => {
   const tax = useSelector((state) => state.cart.tax);
   const total = roundedNumber(useSelector((state) => state.cart.totalAmount)); //total amount after tax
 
+  const handleMoveToWishlist = (album) => {
+    dispatch(reduceItem({ album, quantity: 1 }));
```

Comment:
This should probably remove the item altogether from the order sidebar/cart so we should reduce it by the album's current quantity in the cart